.bazelrc

```diff
@@ -129,6 +128,13 @@ build:auto-release --config=remote
 build:auto-release --remote_instance_name=buildbuddy-io/buildbuddy/auto-release
 build:auto-release -c opt
 
+# On multi-user machines, specify HERMETIC_CC_TOOLCHAIN_CACHE_PREFIX in your
+# repo_env in user.bazelrc to avoid permissions errors.
+build:static --repo_env BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
+build:static --extra_toolchains=@zig_sdk//toolchain:linux_amd64_musl,@zig_sdk//toolchain:linux_arm64_musl
```

Comment:
Did you have a chance to test this on an ARM64 machine?
If not, I will make a conservative call to leave it out until we can verify that it works.

Reply:
Yeah, I did not. I just added this in case someone might find it useful, but it's fair that we're not currently supporting an arm64 static build.